source/src/in/joind/EventListFragment.java

```diff
@@ -53,25 +54,28 @@ public void onAttach(Activity activity) {
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         ViewGroup viewGroup = (ViewGroup) super.onCreateView(inflater, container, savedInstanceState);
 
-        View lvOld = viewGroup.findViewById(android.R.id.list);
+        if (viewGroup != null) {
```

Comment:
As for the Action Bar, Lint was complaining that viewGroup could be Null, so I changed the code.